### PR TITLE
Add pinball loss function for quantile regression

### PR DESCRIPTION
**Pull request by frexvahi**
Also added some docstrings for some of the other loss functions

### CODE REVIEW

**Comment by frexvahi** [comment]
Is it also worth adding some standard quantiles as aliases (e.g. pinball75 for 75th percentile)?

**Comment by frexvahi** [comment]
As an associated metric it might be worth showing what proportion of `y_pred` values are greater than the associated `y_true` value. Should I add that feature to this pull request?

**Comment by fchollet** [comment]
Thanks for the PR! I believe this loss is relatively niche (e.g. not included in sklearn), especially within the field of deep learning (not included in any other DL framework). As such, we should not merge it into core, and I recommend that you open a PR in Keras-contrib instead. 

> Is it also worth adding some standard quantiles as aliases (e.g. pinball75 for 75th percentile)?

Since the API allows for specifying the percentile directly, this is not necessary.
